src/StaticCaching/Replacers/NoCacheReplacer.php

```diff
@@ -78,8 +79,12 @@ private function modifyFullMeasureResponse(Response $response)
         $contents = $response->getContent();
 
         if ($cacher->shouldOutputJs()) {
-            $js = $cacher->getNocacheJs();
-            $contents = str_replace('</body>', '<script type="text/javascript">'.$js.'</script></body>', $contents);
+            $firstLink = Str::position($contents, '<link');
+            $firstScript = Str::position($contents, '<script');
```

Comment:
Maybe add a `Str::position($contents, '<body')` too - just to cover the edge case that there are no links or script tags... it can happen. I think this will ensure the tests pass too.

Reply:
Good catch! Haven't got around to looking at the failed tests yet.